src/pages/unseoJang/Main/Main.scss

```
@@ -1,13 +1,21 @@
 /*nav*/
-.main-wrap {
+.main-wrap.other {
+  * {
+    font-size: 12px;
+    color: #777;
+  }
```

가독성을 위해 다른 셀렉터들끼리는 한 줄 공백 주세요!

---

* 모든 요소들에 적용시키기 위해 와일드카드를 사용하신 것 같은데 부작용이 있을 수 있고 원치않은 요소들에도 속성을 미칠 수 있습니다!

```suggestion
.main-wrap.other{
  font-size:12px;
  color:#777
}
```
로 해도 자식 태그들에 자연스럽게 스타일이 상속되기 때문에 굳이 * 사용하실 필요 없어보입니다!